javascript/angular-todo/src/environments/environment.ts

```diff
@@ -1,10 +1,10 @@
 export const environment = {
-   AM_URL: 'undefined',
-   REALM_PATH: 'undefined',
-   WEB_OAUTH_CLIENT: 'undefined',
-   JOURNEY_LOGIN: 'undefined',
-   JOURNEY_REGISTER: 'undefined',
-   API_URL: 'undefined',
+   SERVER_URL: 'https://openam-sdks.forgeblocks.com/am',
+   REALM_PATH: 'alpha',
+   WEB_OAUTH_CLIENT: 'CentralLoginOAuthClient-',
```

Comment:
Does the client have a `-` at the end or is that a mistake?

Reply:
It does